app/src/main/java/tv/vizbee/demo/activity/MainActivity.kt

```diff
@@ -47,10 +80,88 @@ class MainActivity : AppCompatActivity(),
             .commit()
     }
 
+    override fun showUserLoginFragment() {
+        val tag = UserLoginFragment::class.java.simpleName
+
+        if (supportFragmentManager.findFragmentByTag(tag) != null) {
+            // Do not add fragment if its already on the stack
+            return
+        }
+
+        supportFragmentManager.beginTransaction()
+            .replace(R.id.container, UserLoginFragment.newInstance(), tag)
+            .addToBackStack(null)
+            .commit()
+    }
+
+    override fun popBackStack() {
+        supportFragmentManager.popBackStack()
+    }
+
     override fun onCreateOptionsMenu(menu: Menu): Boolean {
         // Inflate the menu; this adds items to the action bar if it is present.
         menuInflater.inflate(R.menu.menu_main, menu)
         return true
     }
 
+    override fun onPrepareOptionsMenu(menu: Menu): Boolean {
+        val item = menu.findItem(R.id.account)
+        val context = VizbeeWrapper.context?.get()
```

Comment:
Are we in activity and asking vizbee wrapper to give us context?

Reply:
Changed.